app/src/pages/inside/common/filterEntitiesGroups/stepLevelEntities.jsx

```diff
@@ -65,11 +65,16 @@ import {
   ENTITY_PATTERN_NAME,
   ENTITY_RETRY,
   ENTITY_ATTRIBUTE,
+  ENTITY_NEW_FAILURE,
 } from 'components/filterEntities/constants';
 import { defectTypesSelector, patternsSelector } from 'controllers/project';
 import { launchIdSelector } from 'controllers/pages';
-import { levelSelector } from 'controllers/testItem';
+import { getQueryNamespace, levelSelector, PROVIDER_TYPE_LAUNCH } from 'controllers/testItem';
 import { pageEventsMap } from 'components/main/analytics';
+import { connectRouter } from 'common/utils';
+import { querySelector } from 'controllers/administrate/allUsers';
```

Comment:
Are you sure that you should use this selector from `administrate/allUsers`?
You can use `queryParametersSelector` from `testItem/selectors` and provide proper namespace directly in it.